### DISCUSSION TITLE
How to use the model in the package itself on rakit framework?

### DISCUSSION THREAD

**Discussion by wznuhidayat** [esyede/rakit#33] [Diskusi Umum] [answered]
saya membuat package bernama admin 

didalamnya ada model, controller, view dll

posisi saya sedang bermain logic di controller.. bagaimana cara saya memanggil model yang sudah saya buat didalam package tersebut?

**Comment by esyede** [comment, marked as answer]
> bagaimana cara saya memanggil model yang sudah saya buat didalam package tersebut?


Jika model nya tidak bernamespace, maka bisa langsung dipanggil sesuai nama kelasnya, tetapi jika kelasnya memiliki namespace, maka harus di load dulu via file `boot.php`

Contohnya seperti paket berikut ya:
https://github.com/esyede/access
https://github.com/esyede/access/blob/main/boot.php

**Reply by wznuhidayat** [reply]
> > bagaimana cara saya memanggil model yang sudah saya buat didalam package tersebut?
> 
> Jika model nya tidak bernamespace, maka bisa langsung dipanggil sesuai nama kelasnya, tetapi jika kelasnya memiliki namespace, maka harus di load dulu via file `boot.php`
> 
> Contohnya seperti paket berikut ya: https://github.com/esyede/access https://github.com/esyede/access/blob/main/boot.php

thank you 

nice framework mas
